checkov/cloudformation/parser/__init__.py

```diff
@@ -43,4 +44,11 @@ def parse(filename: str) -> Union[Tuple[dict_node, List[Tuple[int, str]]], Tuple
     except YAMLError as err:
         pass
 
+    if template:
+        resources = template.get(TemplateSections.RESOURCES.value, None)
+        if resources:
+            if '__startline__' in resources:
+                resources.pop('__startline__')
+            if '__endline__' in resources:
+                resources.pop('__endline__')
```

Comment:
```suggestion
            if '__startline__' in resources:
                del resources['__startline__']
            if '__endline__' in resources:
                del resources['__endline__']
```
is performance wise better, `pop` is usually used to actually do something with the removed key-value or you could remove the `if` condition and directly use `pop`, like `resources.pop('__startline__', None)`